test/WebSites/ContentNegotiationWebSite/Controllers/FallbackOnTypeBasedMatchController.cs

```diff
@@ -40,7 +40,7 @@ public IActionResult FallbackGivesNoMatch(int input)
         public IActionResult OverrideTheFallback_UsingCustomFormatters(int input)
         {
             var objectResult = new ObjectResult(input);
```

Comment:
There should be a test case which tests the overriding behavior of options and the object result (the later overrides former).